### PR TITLE
Feature/separate testing data

### PR DESCRIPTION
**Pull request by RMartin0717**
# Pull Request Template
## Description

Fixed recipe testing data and refactored/debugged user class tests 

Fixes # (issue)
## Type of change
- [x] Bug fix (non-breaking change which fixes an issue)
- [x] Refactoring
## Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [n/a] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] Any dependent changes have been merged and published in downstream modules
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

File: test/recipe-testing-data.js [hunk 1]
```diff
@@ -5,76 +5,87 @@ const recipeTestingData = [
     "image": "https://spoonacular.com/recipeImages/595736-556x370.jpg",
     "ingredients": [
       {
+        "name": "all purpose flour",
```
Comment (michannstoner) [inline test/recipe-testing-data.js:8]: Good catch on this, Riley!

File: src/user.js [hunk 2]
```diff
@@ -32,12 +31,14 @@ class User {
...
-      return recipe.name.includes(searchWord)
-      || recipe.ingredients.find(ingredient => {
-        return ingredient.name.includes(searchWord)
-      });
-    });
+    const checkName = this.favoriteRecipes.filter(recipe => recipe.name.includes(searchWord))
+
+    const checkIng = this.favoriteRecipes.filter(recipe => {
+       const checkNestedIngredients = recipe.ingredients.find(ingredient => ingredient.name.includes(searchWord))
+       return checkNestedIngredients
```
Comment (bhohnco) [inline src/user.js:38]: nice work adding this

File: test/recipe-testing-data.js [hunk 3]
```diff
@@ -123,83 +134,95 @@ const recipeTestingData = [
...
       {
+        "name": "soy sauce",
         "id": 16124,
         "quantity": {
           "amount": 1,
           "unit": "tablespoon"
         }
       },
       {
+        "name": "sriracha",
```
Comment (bhohnco) [inline test/recipe-testing-data.js:225]: thanks for cleaning this up

**Review by bhohnco — APPROVED** [review]

**Review by michannstoner — APPROVED** [review]
Looks good, thank you for doing this!

**Pull request merged**